File: Codigos em C/Lista 07-24/1. Laboratorio.c
 #include <stdio.h>
#include <string.h>
int main() {

int codigo, quantidade;
char A[10];
float preco;

printf ("\n");
printf ("            Laboratorio: \n");
printf ("(501) Hemograma completo = 28.97 R$ \n");
printf ("(502) Colesterol         = 22.87 R$ \n");
printf ("(503) Creatinina e ureia = 23.74 R$ \n");
printf ("(504) Glicemia           = 19.74 R$ \n");
printf ("(505) Triglicerideos     = 26.22 R$ \n");
printf ("(506) Transaminases      = 33.47 R$ \n");
printf ("(507) Acido urico        = 29.99 R$ \n");

    printf("Digite o Codigo referente ao exame: \n");
    scanf("%i", &codigo);
    printf("Vc possui planos digite S para sim e N para nao: \n");
    scanf ("%s", &A);

switch (codigo) {
    case 501:
    if (strcmp (A, "N") == 0){
        printf ("voce escolheu Hemograma completo 1\n");
        preco = 28.97;
        printf ("O valor total eh %.2f R$", preco);
    }
    else{
    printf ("voce escolheu Hemograma completo 2\n");
    preco = 28.97-(28.97 * 15/100);
    printf ("O valor total eh %.2f R$", preco);

    return 1;
    }
    break;

    case 502:
        if (strcmp (A, "N") == 0){
        printf ("voce escolheu Colesterol\n");
    preco = 22.87;
    printf ("O valor total eh %.2f R$", preco);
    }
    else{
    printf ("voce escolheu Colesterol 2\n");
    preco = 22.87-(22.87 * 15/100);
    printf ("O valor total eh %.2f R$", preco);
    return 1;
    }
    break;

    case 503:
        if (strcmp (A, "N") == 0){
        printf ("voce escolheu Creatinina e ureia\n");
    preco =  23.74;
    printf ("O valor total eh %.2f R$", preco);
     }
    else{
             printf ("voce escolheu Creatinina e ureia 2\n");
    preco = 23.74-  (23.74 * 15/100);
    printf ("O valor total eh %.2f R$", preco);
    return 1;
    }
    break;
    case 504:
        if (strcmp (A, "N") == 0){
        printf ("voce escolheu Glicemia\n");
    preco =  19.74;
    printf ("O valor total eh %.2f R$", preco);
         }
    else{
             printf ("voce escolheu Glicemia 2\n");
    preco = 19.74- (19.74 * 15/100);
    printf ("O valor total eh %.2f R$", preco);
    return 1;
    }
    break;
    case 505:
        if (strcmp (A, "N") == 0){
        printf ("voce escolheu Triglicerideos\n");
    preco =  26.22;
    printf ("O valor total eh %.2f R$", preco);
             }
    else{
             printf ("voce escolheu Triglicerideos 2\n");
    preco = 26.22-(26.22 * 15/100);
    printf ("+ (O valor total eh %.2f R$", preco);
    return 1;
    }
    break;
    case 506:
        if (strcmp (A, "N") == 0){
        printf ("voce escolheu Transaminases\n");
    preco =  33.47;
    printf ("O valor total eh %.2f R$", preco);
     }
    else{
             printf ("voce escolheu Transaminases 2\n");
    preco = 33.47-(33.47 * 15/100);
    printf ("O valor total eh %.2f R$", preco);
    return 1;
    }
    break;
     case 507:
         if (strcmp (A, "N") == 0){
        printf ("voce escolheu acido urico\n");
    preco =  29.99;
    printf ("O valor total eh %.2f R$", preco);
     }
    else{
             printf ("voce escolheu acido urico 2\n");
    preco =  29.99- (29.99 * 15/100);
    printf ("O valor total eh %.2f R$", preco);
    return 1;
    }
    break;
default:
  printf("Codigo invalido.\n");
 break;
    }
printf("\n");
    return 0;
}

File: Conversão para Python/Lista 07-24/1. Laboratorio.py

print("            Laboratorio: ")
print("(501) Hemograma completo = 28.97 R$")
print("(502) Colesterol         = 22.87 R$")
print("(503) Creatinina e ureia = 23.74 R$")
print("(504) Glicemia           = 19.74 R$")
print("(505) Triglicerideos     = 26.22 R$")
print("(506) Transaminases      = 33.47 R$")
print("(507) Acido urico        = 29.99 R$")

codigo = int(input("Digite o Codigo referente ao exame: "))
A = input("Você possui planos? Digite 'S' para sim e 'N' para não: ")

preco = 0

if codigo == 501:
    if A == "N":
        print("Você escolheu Hemograma completo 1")
        preco = 28.97
        print(f"O valor total é {preco:.2f} R$")
    else:
        print("Você escolheu Hemograma completo 2")
        preco = 28.97 - (28.97 * 15/100)
        print(f"O valor total é {preco:.2f} R$")
elif codigo == 502:
    if A == "N":
        print("Você escolheu Colesterol")
        preco = 22.87
        print(f"O valor total é {preco:.2f} R$")
    else:
        print("Você escolheu Colesterol 2")
        preco = 22.87 - (22.87 * 15/100)
        print(f"O valor total é {preco:.2f} R$")
elif codigo == 503:
    if A == "N":
        print("Você escolheu Creatinina e ureia")
        preco = 23.74
        print(f"O valor total é {preco:.2f} R$")
    else:
        print("Você escolheu Creatinina e ureia 2")
        preco = 23.74 - (23.74 * 15/100)
        print(f"O valor total é {preco:.2f} R$")
elif codigo == 504:
    if A == "N":
        print("Você escolheu Glicemia")
        preco = 19.74
        print(f"O valor total é {preco:.2f} R$")
    else:
        print("Você escolheu Glicemia 2")
        preco = 19.74 - (19.74 * 15/100)
        print(f"O valor total é {preco:.2f} R$")
elif codigo == 505:
    if A == "N":
        print("Você escolheu Triglicerideos")
        preco = 26.22
        print(f"O valor total é {preco:.2f} R$")
    else:
        print("Você escolheu Triglicerideos 2")
        preco = 26.22 - (26.22 * 15/100)
        print(f"O valor total é {preco:.2f} R$")
elif codigo == 506:
    if A == "N":
        print("Você escolheu Transaminases")
        preco = 33.47
        print(f"O valor total é {preco:.2f} R$")
    else:
        print("Você escolheu Transaminases 2")
        preco = 33.47 - (33.47 * 15/100)
        print(f"O valor total é {preco:.2f} R$")
elif codigo == 507:
    if A == "N":
        print("Você escolheu ácido úrico")
        preco = 29.99
        print(f"O valor total é {preco:.2f} R$")
    else:
        print("Você escolheu ácido úrico 2")
        preco = 29.99 - (29.99 * 15/100)
        print(f"O valor total é {preco:.2f} R$")
else:
    print("Código inválido.")

